client/components/account-balances/style.scss

```diff
@@ -35,3 +35,29 @@
 	border-top: 1px solid $gray-200;
 	padding: 24px;
 }
+
+.wcpay-account-balances__balances {
+	.wcpay-account-balances__balances__item__tooltip {
+		line-height: 20px;
+	}
+
+	.wcpay-account-balances__balances__item__tooltip a {
+		text-decoration: none;
+	}
+
+	.wcpay-account-balances__balances-total-balance-tooltip-notice {
+		margin-top: 10px;
+		margin-bottom: 10px;
+
+		&.components-notice {
+			padding: 8px;
+		}
+
+		.components-notice__content {
+			font-size: 12px;
+			white-space: nowrap;
+			color: #054b7a;
```

Comment:
Nitpick: do we have a color code for this already defined to be used?

Reply:
```suggestion
			color: $wp-blue-70;
```

I think `$wp-blue-70` is the closest match, but haven't looked at Figma design.

<img width="132" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/fba78f5f-0b30-4029-a546-1f3b734e386a">

Reply:
@Jinksi can you please point me the file where all the CSS variables are defined?

Reply:
> I think $wp-blue-70 is the closest match

Looks like it.

Asked further about this here: p1709146807215419-slack-C02BW3Z8SHK

Reply:
https://github.com/Automattic/woocommerce-payments/blob/835df7f966da292b69e3a3b16fdfe41ce04a8611/client/stylesheets/abstracts/_colors.scss#L29

Also https://github.com/Automattic/color-studio/blob/master/dist/color-variables.scss are imported and used throughout WooPayments.